docs/notes/setup.md

```diff
@@ -56,6 +56,13 @@ On Windows, you need to do:
 On Mac, you need to do:
 	�<installerexe>/Contents/MacOS/installer� install noui *options*
 On both platforms, the result of the installation will be written to the console.
+# Command-line uninstallation
+It is possible to uninstall LiveCode from the command-line on Windows and Linux, from LiveCode *6.7.8* and *7.1.1*. When invoked in this fashion, no GUI will be displayed.
```

Comment:
I don't think you need to include the explicit version here -- the fact that this information is in a particular version's release notes implies that the behaviour is available for that version.